pageserver/src/tenant/storage_layer.rs

```diff
@@ -158,41 +159,82 @@ impl LayerAccessStatFullDetails {
 }
 
 impl LayerAccessStats {
-    pub(crate) fn for_loading_layer(status: LayerResidenceStatus) -> Self {
-        let new = LayerAccessStats(Mutex::new(LayerAccessStatsLocked::default()));
-        new.record_residence_event(status, LayerResidenceEventReason::LayerLoad);
-        new
+    /// Create an empty stats object.
+    ///
+    /// The caller is responsible for recording a residence event
+    /// using [`record_residence_event`] before calling `latest_activity`.
+    /// If they don't, [`latest_activity`] will return `None`.
+    pub(crate) fn empty_will_record_residence_event_later() -> Self {
+        LayerAccessStats(Mutex::new(LayerAccessStatsLocked::default()))
```

Comment:
Just FYI, this could be simplified as Mutex's and RefCells and pretty much all types with Default impl also use the inner types Default::default().

Most terse I'd be fine here:

```
LayerAccessStats(Default::default())
```

But in general we hope to avoid fully generic, so

```suggestion
        LayerAccessStats(Mutex::default())
```

Would work just as well. But the current is fine as well.

Reply:
TBH I never got why we don't use `Default::default()`